### PR TITLE
2017 PDF sets using systematics module for MadGraph 2.5.5

### PR DESCRIPTION
**Pull request by kdlong**
This is probably not quite ready to be merged, but I'm starting the pull request so it can be referenced and so people can start to comment.

### CODE REVIEW

**Comment by perrozzi** [comment]
any news here?

File: bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat [hunk 1]
```diff
@@ -92,20 +92,20 @@
 #*********************************************************************** 
 # Reweight variables for scale dependence and PDF uncertainty          *
 #***********************************************************************
- 1.0, 2.0, 0.5 = rw_rscale ! muR factors to be included by reweighting
- 1.0, 2.0, 0.5 = rw_fscale ! muF factors to be included by reweighting
+ 1.0 = rw_rscale ! muR factors to be included by reweighting
+ 1.0 = rw_fscale ! muF factors to be included by reweighting
  True = reweight_scale ! Reweight to get scale variation using the
```
Comment (bendavid) [inline bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat]: Would it make sense to have False here as well?

**Comment by bendavid** [comment]
This pull request should probably also remove the installation/compilation of syscalc from the main script.

File: bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat (hunk 1 above)
Comment (kdlong) [inline bin/MadGraph5_aMCatNLO/cards/examples/wplustest_5f_NLO/wplustest_5f_NLO_run_card.dat]: Annoyingly 1 of the 2 between reweight_PDF and reweight_scale have to be set to true for the weights to be stored. I asked Olivier about this and it seemed intentional

**Comment by kdlong** [comment]
@bendavid agreed, done now. Should also update the PDFs (and maybe add a way for users to customize the PDFs through cards)

**Comment by kdlong** [comment]
I'm not sure if the PDF list is yet the "final" one but otherwise I think this can be merged.

**Pull request merged**